src/Card/index.scss

```diff
@@ -353,7 +353,15 @@ a.pgn__card {
       width: var(--pgn-size-card-logo-width);
       height: var(--pgn-size-card-logo-height);
       border-radius: var(--pgn-size-card-border-radius-logo);
-      box-shadow: var(--pgn-elevation-box-shadow-level-1);
+      box-shadow:
+        var(--pgn-elevation-box-shadow-level-1-1-offset-y)
+        var(--pgn-elevation-box-shadow-level-1-1-offset-x)
+        var(--pgn-elevation-box-shadow-level-1-1-blur)
+        var(--pgn-elevation-box-shadow-level-1-1-color),
+        var(--pgn-elevation-box-shadow-level-1-2-offset-y)
+        var(--pgn-elevation-box-shadow-level-1-2-offset-x)
+        var(--pgn-elevation-box-shadow-level-1-2-blur)
+        var(--pgn-elevation-box-shadow-level-1-2-color);
```

Comment:
In this case, we could something similar as above comment (i.e., create an abstracted CSS variable that consolidates each of these independent CSS variables into one).

However, something I'm still contemplating (curious for your input!) is whether such a abstracted CSS variable should be manually defined here more like an CSS variable, e.g. `--pgn-elevation-card-box-shadow`, which would contain the specific parts/layered box-shadow as exists currently.

While this would give theme authors the ability to override box-shadow based on `--pgn-elevation-card-box-shadow` OR each individual CSS variable part, I might wonder if there should be the abstracted group of `--pgn-elevation-box-shadow-level-1` created for each elevation level out-of-the-box, whereby this `box-shadow` could continue to use `--pgn-elevation-box-shadow-level-1`.

The challenge here is how/where to generate the abstracted CSS variable that acts as the grouping for the independent CSS variable parts. 

For example, is there a way to get `--pgn-elevation-box-shadow-level-1` programmatically generated during `build-tokens`, or will we need to manually create these abstraction variables ourselves? 🤔

Reply:
My mind goes to either:
* Manually creating the abstraction instead of trying to create the abstraction based on the expanded set

or

* Making the expanded set based on the abstraction (adding `--pgn-elevation-box-shadow-level-1` automatically adds all the expanded ones).

That being said, I think both trying to create the abstraction from the expanded or create the expanded from the abstraction will add complexity.

I believe handling it manually is likely the solution that will lead to the fewest snags.

Reply:
This is also where I'm leaning (i.e., manually created abstractions), as I'm not sure whether `style-dictionary` actually could support doing something like this programmatically.

Regarding _where_ we manually define these CSS variable abstractions (e.g., `--pgn-elevation-box-shadow-level-1`), it probably shouldn't go in the generated output from `style-dictionary` (i.e., the `variables.css` files) as those should remain only for automated output to prevent overriding of custom changes when running `build-tokens`.

Maybe we define a new file under `core` and `themes/light` to define these abstractions for more "global" variables (e.g., something like `abstraction-variables.css`), which gets included in the `index.css` where we currently import `variables.css`).

If the abstraction is specific to an individual component (e.g., `Annotation`'s `filter` style property), the abstraction could be defined closer to where it's used as described in an [above comment](https://github.com/openedx/paragon/pull/3222#discussion_r1759732065).

Reply:
> something like `abstraction-variables.css`

I like this a lot!

> the abstraction could be defined closer to where it's used as described

I'm torn on this idea. On one side it feels like a cleaner way to structure things, but on the other side it feels like it'd be harder for someone trying to make overrides to know all the abstractions available to them. Maybe there's a way we can have our cake and eat it too here I'm missing :thinking:

Reply:
Thanks, added changes in the last commit, please take a look

Reply:
That looks great! I'm still not 100% sure what the long term plan for where abstractions live should be, but for now with `abstraction-variables.css` being ~300 lines having everything directly in there makes sense. Thanks for putting it together!